src/suggestions.js

```diff
@@ -70,22 +70,30 @@ Suggestions.prototype.handleKeyUp = function(keyCode) {
 Suggestions.prototype.handleKeyDown = function(e) {
   switch (e.keyCode) {
     case 13: // ENTER
-    case 9: // TAB
+      if (this.list.active === undefined) {
+        return
+      }
       if (!this.list.isEmpty()) {
         if (this.list.isVisible()) {
           e.preventDefault();
         }
         this.value(this.list.items[this.list.active].original);
         this.list.hide();
       }
+    case 9: // TAB
```

Comment:
Do we need a `break` here?
```suggestion
    break;
    case 9: // TAB
```

Reply:
Good call. Updated